### DISCUSSION TITLE
DM3 OSC Specs might be helpful

### DISCUSSION THREAD

**Discussion by hansSchall** [BrenekH/yamaha-rcp-docs#3] [General]
Found this: https://fr.yamaha.com/files/download/other_assets/2/2063222/DM3_osc_specs_v100_en.pdf

It uses UDP OSC as transport layer, but the commands seem to be the same.

The document includes all commands, comments and possible values.

**Comment by BrenekH** [comment]
Thanks so much! That's very helpful!
